doc/source/structures/collections/list.rst

```diff
@@ -41,6 +41,9 @@ Structure
           - Type
           - Description
 
+        * - All suffixes of :struct:`Enumerable`
+          -
+          - :struct:`List` objects are a type of :struct:`Enumerable`
```

Comment:
Technically, it's a kind of `Collection` now, and a `Collection`, in turn, is a kind of `Enumerator`.  Thus the 'CLEAR' suffix is part of `Collection`.  But I'm not sure we want this to jump through two levels of link before we get to LENGTH, CONTAINS, And so on.